src/pages/gateway/headers.md

```diff
@@ -124,6 +124,65 @@ Mesh owners can use the `responseConfig.headers` object to add response headers.
     }
 ```
 
+#### Forward headers
```

Comment:
Why "forward" ? Maybe "Return header values"?

Reply:
i believe "Forward headers" is the correct terminology ([e.g.](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Forwarded)), as the headers are forwarded from the source. @of0x would it make more sense for this section to be named "Return header values"?

Reply:
Based on your link, I prefer "Forwarded" over "Forward".  Maybe insert "forwarded" between "return" and "header" in the sentence below.

Reply:
Yes, forward is correct. The mesh is an intermediary between the various sources and the requesting client, so it is forwarding the headers to the client.